jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibExtensionTest.java

```diff
@@ -335,125 +324,122 @@ public void testSkaffold() {
+        .containsExactly("/foo/bar", "707", "/baz", "456")
+        .inOrder();
 
     System.setProperty("jib.dockerClient.executable", "test-exec");
-    Assert.assertEquals(
-        Paths.get("test-exec"), testJibExtension.getDockerClient().getExecutablePath());
+    assertThat(testJibExtension.getDockerClient().getExecutablePath())
+        .isEqualTo(Paths.get("test-exec"));
     System.setProperty("jib.dockerClient.environment", "env1=val1,env2=val2");
-    Assert.assertEquals(
-        ImmutableMap.of("env1", "val1", "env2", "val2"),
-        testJibExtension.getDockerClient().getEnvironment());
+    assertThat(testJibExtension.getDockerClient().getEnvironment())
+        .containsExactly("env1", "val1", "env2", "val2")
+        .inOrder();
 
     // Absolute paths
     System.setProperty("jib.outputPaths.digest", "/digest/path");
-    Assert.assertEquals(
-        Paths.get("/digest/path").toAbsolutePath(),
-        testJibExtension.getOutputPaths().getDigestPath());
+    assertThat(testJibExtension.getOutputPaths().getDigestPath())
+        .isEqualTo(Paths.get("/digest/path").toAbsolutePath());
     System.setProperty("jib.outputPaths.imageId", "/id/path");
-    Assert.assertEquals(
-        Paths.get("/id/path").toAbsolutePath(), testJibExtension.getOutputPaths().getImageIdPath());
+    assertThat(testJibExtension.getOutputPaths().getImageIdPath())
+        .isEqualTo(Paths.get("/id/path").toAbsolutePath());
     System.setProperty("jib.outputPaths.tar", "/tar/path");
-    Assert.assertEquals(
-        Paths.get("/tar/path").toAbsolutePath(), testJibExtension.getOutputPaths().getTarPath());
+    assertThat(testJibExtension.getOutputPaths().getTarPath())
+        .isEqualTo(Paths.get("/tar/path").toAbsolutePath());
     // Relative paths
     System.setProperty("jib.outputPaths.digest", "digest/path");
-    Assert.assertEquals(
-        fakeProject.getProjectDir().toPath().resolve(Paths.get("digest/path")),
-        testJibExtension.getOutputPaths().getDigestPath());
+    assertThat(testJibExtension.getOutputPaths().getDigestPath())
+        .isEqualTo(fakeProject.getProjectDir().toPath().resolve("digest/path"));
     System.setProperty("jib.outputPaths.imageId", "id/path");
-    Assert.assertEquals(
-        fakeProject.getProjectDir().toPath().resolve(Paths.get("id/path")),
-        testJibExtension.getOutputPaths().getImageIdPath());
+    assertThat(testJibExtension.getOutputPaths().getImageIdPath())
+        .isEqualTo(fakeProject.getProjectDir().toPath().resolve("id/path"));
     System.setProperty("jib.outputPaths.tar", "tar/path");
-    Assert.assertEquals(
-        fakeProject.getProjectDir().toPath().resolve(Paths.get("tar/path")),
-        testJibExtension.getOutputPaths().getTarPath());
+    assertThat(testJibExtension.getOutputPaths().getTarPath())
+        .isEqualTo(fakeProject.getProjectDir().toPath().resolve("tar/path"));
+    System.setProperty("jib.configurationName", "myRuntimeClasspath");
```

Comment:
Since `runtimeClasspath` is the default, how about this?
```suggestion
    System.setProperty("jib.configurationName", "myConfiguration");
```